src/Microsoft.Diagnostics.Monitoring.RestServer/MetricsOptions.cs

```diff
@@ -18,24 +20,40 @@ namespace Microsoft.Diagnostics.Monitoring.RestServer
     /// </summary>
     public class MetricsOptions
     {
+        [Description("Enable or disable metrics collection. Defaults to true.")]
```

Comment:
Is the value of DefaultValue picked up by IntelliSense? If so, then we probably don't need to say that it defaults to true.